File: app/src/main/java/view/pager/recycler/utils/Check.java
package view.pager.recycler.utils;

/**
 * Created by harneev on 20/09/16.
 */

public class Check {

    public static boolean isEmpty(String string) {
        try {

            if (string == null || string.trim().length() == 0 || string.length() == 0 || string.isEmpty()
                    || string.equalsIgnoreCase("null")) {
                return true;
            }

        } catch (Exception e) {
            return false;
        }

        return false;
    }

    public static boolean isEmpty(Integer value) {
        try {

            if (value == null || value == -1 || value == 0) {
                return true;
            }

        } catch (Exception e) {
            return false;
        }

        return false;
    }
}

File: app/src/main/java/view/pager/recycler/utils/Check.kt
package view.pager.recycler.utils

object Check {

    fun isEmpty(string: String?): Boolean {
        try {

            if (string == null || string.trim { it <= ' ' }.isEmpty() || string.isEmpty() || string.isEmpty()
                    || string.equals("null", ignoreCase = true)) {
                return true
            }

        } catch (e: Exception) {
            return false
        }

        return false
    }
}
